app/Support/Yaml.php

```diff
@@ -46,8 +45,6 @@ public function loadToConfig( $path, $configKey ): Collection {
             $loaded = array_replace_recursive( ...$loaded );
```

Comment:
🌮 I know it's not a part of this, but I never knew about `array_replace_recursive()`